Packs/WootCloud/Integrations/WootCloud/README.md

```diff
@@ -0,0 +1,465 @@
+| WootCloud.PacketAlert.destination.longitude | Number | destination longitude | 
+| WootCloud.PacketAlert.destination.mac | String | destination mac address | 
+| WootCloud.PacketAlert.destination.network | String | destination network | 
+| WootCloud.PacketAlert.destination.port | Number | destination port | 
+| WootCloud.PacketAlert.destination.state | String | destination state | 
+| WootCloud.PacketAlert.destination.subnet | String | destination subnet | 
+| WootCloud.PacketAlert.destination.time_zone | String | destination time zone | 
+| WootCloud.PacketAlert.destination.zip | String | destination zip | 
+| WootCloud.PacketAlert.destination.inferred.device_id | String | destination inferred device ID | 
+| WootCloud.PacketAlert.destination.inferred.asset | String | destination inferred asset | 
+| WootCloud.PacketAlert.destination.inferred.managed | Number | destination inferred managed | 
+| WootCloud.PacketAlert.destination.inferred.category | String | destination inferred category | 
+| WootCloud.PacketAlert.destination.inferred.control | String | destination inferred control | 
+| WootCloud.PacketAlert.destination.inferred.host_name | String | destination inferred host name | 
+| WootCloud.PacketAlert.destination.inferred.os | String | destination inferred OS | 
+| WootCloud.PacketAlert.destination.inferred.os_version | String | destination inferred OS version | 
+| WootCloud.PacketAlert.destination.inferred.ownership | String | destination inferred ownership | 
+| WootCloud.PacketAlert.destination.inferred.total_risk | Number | destination inferred total risk score | 
+| WootCloud.PacketAlert.destination.inferred.type | String | destination inferred type | 
+| WootCloud.PacketAlert.destination.inferred.username | String | destination inferred username | 
+| WootCloud.PacketAlert.destination.inferred.managed_info.host_name | String | destination inferred managed info hostname | 
+| WootCloud.PacketAlert.payload | String | payload | 
+| WootCloud.PacketAlert.http.hostname | String | http hostname | 
+| WootCloud.PacketAlert.http.http_method | String | http methon | 
+| WootCloud.PacketAlert.http.http_user_agent | String | http user agent | 
+| WootCloud.PacketAlert.http.length | Number | http length | 
+| WootCloud.PacketAlert.http.protocol | String | http protocol | 
+| WootCloud.PacketAlert.http.redirect | String | http redirect | 
+| WootCloud.PacketAlert.http.http_refer | String | http referal | 
+| WootCloud.PacketAlert.http.status | Number | http status code | 
+| WootCloud.PacketAlert.http.url | String | http url | 
+| WootCloud.PacketAlert.type | String | http type | 
+| WootCloud.PacketAlert.group | String | group | 
+| WootCloud.PacketAlert.subtype | String | subtype | 
+| WootCloud.PacketAlert.title | String | title | 
+| WootCloud.PacketAlert.description | String | description | 
+| WootCloud.PacketAlert.references | String | references | 
+
+
+#### Command Example
+``` ```
```

Comment:
The readme should contain also the commands example and their outputs.
You can use `demisto-sdk generate-docs` with -e flag.
https://github.com/demisto/demisto-sdk/tree/master/demisto_sdk/commands/generate_docs

Reply:
Added command examples to the README